### PR TITLE
refactor: remove extra characters in project installation

### PR DESCRIPTION
**Pull request by RichardBray**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,14 +38,14 @@ This website was created with [Docusaurus](https://v2.docusaurus.io).
 
 ```sh
 # Install dependencies
-$ npm install
+$ npm i
```
Comment (nickmccurdy) [inline README.md:41]: I realize this is a bit of a nitpick but I'd rather keep install as a full word to be easier to learn

**Review by nickmccurdy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RichardBray) [reply README.md:41]: Good point, I will put that back

**Review by nickmccurdy — APPROVED** [review]

**Comment by alexkrolick** [comment]
Oh, that's new. When did npm start supporting the script shorthand?

**Comment by nickmccurdy** [comment]
As far as I'm aware, it's as old as npm itself

**Comment by nickmccurdy** [comment]
Specific scripts like `start` and `test` have been supported for a while, but unlike yarn you still need `npm run` to use other scripts

**Pull request merged**